Django-CoWork-Connect/coapp/migrations/0004_alter_desk_monitor_number_and_more.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 5.0.3 on 2024-05-06 21:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("coapp", "0003_reservation"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="desk",
+            name="monitor_number",
+            field=models.IntegerField(),
+        ),
+        migrations.AlterField(
+            model_name="desk",
+            name="power_socket_count",
+            field=models.IntegerField(),
```

Comment:
Jezeli komentarz nizej jest poprawny to ta migracja jest w ogole nie potrzebna, bo zmiany w modelu sa nic nie znaczace

Reply:
jw.:)